tests/tests/cpld_control_wave

```diff
@@ -0,0 +1,93 @@
+                   dynf024.tile1.nc \
+                   dynf024.tile2.nc \
+                   dynf024.tile3.nc \
+                   dynf024.tile4.nc \
+                   dynf024.tile5.nc \
+                   dynf024.tile6.nc \
+                   RESTART/coupler.res \
+                   RESTART/fv_core.res.nc \
+                   RESTART/fv_core.res.tile1.nc \
+                   RESTART/fv_core.res.tile2.nc \
+                   RESTART/fv_core.res.tile3.nc \
+                   RESTART/fv_core.res.tile4.nc \
+                   RESTART/fv_core.res.tile5.nc \
+                   RESTART/fv_core.res.tile6.nc \
+                   RESTART/fv_srf_wnd.res.tile1.nc \
+                   RESTART/fv_srf_wnd.res.tile2.nc \
+                   RESTART/fv_srf_wnd.res.tile3.nc \
+                   RESTART/fv_srf_wnd.res.tile4.nc \
+                   RESTART/fv_srf_wnd.res.tile5.nc \
+                   RESTART/fv_srf_wnd.res.tile6.nc \
+                   RESTART/fv_tracer.res.tile1.nc \
+                   RESTART/fv_tracer.res.tile2.nc \
+                   RESTART/fv_tracer.res.tile3.nc \
+                   RESTART/fv_tracer.res.tile4.nc \
+                   RESTART/fv_tracer.res.tile5.nc \
+                   RESTART/fv_tracer.res.tile6.nc \
+                   RESTART/phy_data.tile1.nc \
+                   RESTART/phy_data.tile2.nc \
+                   RESTART/phy_data.tile3.nc \
+                   RESTART/phy_data.tile4.nc \
+                   RESTART/phy_data.tile5.nc \
+                   RESTART/phy_data.tile6.nc \
+                   RESTART/sfc_data.tile1.nc \
+                   RESTART/sfc_data.tile2.nc \
+                   RESTART/sfc_data.tile3.nc \
+                   RESTART/sfc_data.tile4.nc \
+                   RESTART/sfc_data.tile5.nc \
+                   RESTART/sfc_data.tile6.nc \
+                   RESTART/MOM.res.nc \
+                   RESTART/iced.2016-10-04-00000.nc \
+                   RESTART/ufs.cpld.cpl.r.2016-10-04-00000.nc \
+                   out_grd.glo_1deg"
+
+export_fv3
+export_cpl
+
+# set component and coupling timesteps
+export DT_ATMOS='450'
+export DT_CICE=${DT_ATMOS}
+export DT_DYNAM_MOM6='900'
+export DT_THERM_MOM6='1800'
+export CPL_SLOW=${DT_THERM_MOM6}
+export CPL_FAST=${DT_ATMOS}
+
+export RESTART_INTERVAL="12"
+export RESTART_N=${RESTART_INTERVAL}
+
+export TASKS=$TASKS_cpl_wwav
+export TPN=$TPN_cpl_wwav
+export INPES=$INPES_cpl_wwav
+export JNPES=$JNPES_cpl_wwav
+export THRD=$THRD_cpl_wwav
+export WRTTASK_PER_GROUP=$WPG_cpl_wwav
+
+export med_petlist_bounds=$MPB_cpl_wwav
+export atm_petlist_bounds=$APB_cpl_wwav
+export ocn_petlist_bounds=$OPB_cpl_wwav
+export ice_petlist_bounds=$IPB_cpl_wwav
+export wav_petlist_bounds=$WPB_cpl_wwav
+
+# nems.configure
+export NEMS_CONFIGURE="nems.configure.cpld_wave.IN"
+export coupling_interval_slow_sec=${CPL_SLOW}
+export coupling_interval_fast_sec=${CPL_FAST}
+
+export CPLWAV='.T.'
+export CPLWAV2ATM='.T.'
+export RT1DEG='.T.'
```

Comment:
In default_vars, we set the default resolution for each component using "OCNRES" etc. I would suggest to create a default WW3 res which is 1deg. You wouldn't need the RT1DEG variable because the default tests would be 1 deg ww3, 1deg ocean, 1deg ice and c96 fv3.  

The current ww3 tests which are all for 1/4 deg would then need to set a non-default resolution. This would be consistent w/ how we vary the resolution across the different tests.

Reply:
@DeniseWorthen Thanks for the suggestions. I modified the test but still get a failure.
Here are my changes:
> adding cpl_dflt_wwav. Do you have any suggestions for the numbers? I assumed the 1deg is lighter than 1/4, and heavier than dflt, so I set the total to 288. What each of these variables stand for? I mean where can I see the definitions?
> I removed export RT1DEG='.T.' 
> I added WAVRES='1.00'